Resources/public/js/components/news/form/main.js

```diff
@@ -40,11 +42,15 @@ define(['underscore', 'jquery', 'text!./form.html'], function(_, $, form) {
         },
 
         render: function() {
-            this.$el.html(this.templates.form({translations: this.translations}));
+            this.$el.html(
+                this.templates.form({translations: this.translations, locale: this.sandbox.sulu.user.locale})
+            );
 
             this.form = this.sandbox.form.create('#news-form');
             this.form.initialized.then(function() {
-                this.sandbox.form.setData('#news-form', this.data || {});
+                this.sandbox.form.setData('#news-form', this.data || {}).then(function() {
+                    this.sandbox.start(this.$el);
```

Comment:
after setting the data, all components in the form will be started by this line of code. after adding this line, you can add more components into the form without changing something in this code.